posthog/api/test/test_decide.py

```diff
@@ -262,6 +264,139 @@ def test_feature_flags_v2_complex(self):
             # third-variant:  20 (100 * 80% * 25% = 20 users)
             # fourth-variant: 20 (100 * 80% * 25% = 20 users)
 
+    def test_feature_flags_v2_override(self):
+        self.user.distinct_id = "static-distinct-id"
+        self.user.save()
+        self.team.app_urls = ["https://example.com"]
+        self.team.save()
+        self.client.logout()
+        Person.objects.create(
+            team=self.team,
+            distinct_ids=[self.user.distinct_id, "not-connonical-distinct-id"],
```

Comment:
"connonical" appears in a lot of places, I suppose this should be "canonical"? 🤔

Reply:
Fixed, and added spell check to my IDE 😅